### PR TITLE
DI-2050: Upgrade spark's version

### PR DESCRIPTION
**Pull request by v44p**
* [x] Updated [CHANGELOG file](https://github.com/jampp/wolfria/blob/develop/CHANGELOG.rst)
* [x] Updated [VERSION file](https://github.com/jampp/wolfria/blob/develop/VERSION.rst)
* [x] Updated versions on `docker-compose.yaml` image names.

### CODE REVIEW

**Review by gonzalezzfelipe — COMMENTED** [review]
Missing updating the version on the `docker-compose.yaml`

**Review by gonzalezzfelipe — APPROVED** [review]
You are only missing the CHANGELOG update, remember to do it before merging

File: CHANGELOG.rst [hunk 1]
```diff
@@ -15,6 +15,15 @@ can be described in a Major section.
 More information can be found HERE:
 
 
+v2.1.2 - unreleased
+===================
```
Comment (lautaroq) [inline CHANGELOG.rst:19]: this is not a fix. The version is 2.2.0

**Review by lautaroq — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,6 +15,15 @@ can be described in a Major section.
 More information can be found HERE:
 
 
+v2.1.2 - unreleased
```
Comment (viniolivieri) [inline CHANGELOG.rst:18]:
```suggestion
v2.2.0 - Unreleased
```

**Review by viniolivieri — APPROVED** [review]
Just the CHANGELOG.

File: CHANGELOG.rst (hunk 1 above)
Comment (v44p) [reply CHANGELOG.rst:19]: Sure!, but is a fix in the spark version, what do you think? we need upgrade to 2.2.0 or represent these fix upgraded in 2.1.2 ?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -30,6 +39,7 @@ v2.1.0 - 2021-11-11
 Added
 -----
 
+* add compatibility and Upgraded Spark to version 3.1.2
```
Comment (lautaroq) [inline CHANGELOG.rst:42]: this line is not correct here

**Review by lautaroq — CHANGES_REQUESTED** [review]

**Review by lautaroq — APPROVED** [review]
Please, squash before merge

**Pull request merged**